viz.yaml

```diff
@@ -256,24 +256,26 @@ publish:
     id: map-section
     publisher: section
     template: simplefigure
-    depends:
+    depends: siteText
       map: map-svg
       gages-js: gages-js
     context:
       id: gage-map
       figure: map
-      caption: states
+      caption: siteText.explanationText1
+		  data: ["siteText.title", "siteText.subtitle"]
```

Comment:
yaml hates tabs, also we will need the template to handle `data` we can get @mwernimont on that as he is changing the templates anyway